src/packages/sdk/src/predefinedGeneratorResolvers.ts

```diff
@@ -1,13 +1,13 @@
-import Debug from '@prisma/debug'
 import chalk from 'chalk'
 import execa from 'execa'
 import fs from 'fs'
 import hasYarn from 'has-yarn'
 import path from 'path'
-import resolvePkg from 'resolve-pkg'
 import { logger } from '.'
 import { getCommandWithExecutor } from './getCommandWithExecutor'
-const debugEnabled = Debug.enabled('generator')
+import { findUpAsync as findUp } from './utils/find'
+
+const load = require
```

Comment:
Why aliasing / hiding require in a const?

Reply:
It's to hide it from bundlers. I find it more beautiful than using `eval` which is slower and is not good for type safety.

Reply:
I'll add a comment